bundles/org.openhab.core.config.discovery/junit5-migrate.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
```

Comment:
I guess that file was not supposed to be committed?

Reply:
No, of course not.